examples/elixir_euler_potential_temperature_baroclinic_instability.jl

```diff
@@ -193,7 +193,7 @@ end
 
     return SVector(du1, du2, du3, du4, du5, zero(eltype(u)))
 end
-equations = CompressibleEulerPotentialTemperatureEquationsWithGravity3D()
+equations = CompressibleEulerPotentialTemperatureEquationsWithGravity3D(1004, 717, 9.81)
```

Comment:
The issue with positional arguments is that it is not completely clear what they mean. Thus, we often use keyword arguments in Trixi.jl since they make the code somewhat self-documenting. Do you think this could help here as well (for these equations and the others)?